daemon/daemon.go

```diff
@@ -1485,6 +1485,34 @@ func CreateDaemonRoot(config *config.Config) error {
 	return setupDaemonRoot(config, realRoot, idMapping.RootPair())
 }
 
+// RemapContainerdNamespaces returns the right containerd namespaces to use:
+// - if they are not already set in the config file
+// -  and the daemon is running with user namespace remapping enabled
+// Then it will return new namespace names, otherwise it will return the existing
+// namespaces
+func RemapContainerdNamespaces(config *config.Config) (string, string, error) {
```

Comment:
Should we look at limiting arguments to what we need (e.g., make this accept just an `idtools.Identity`)? Probably not urgent.